app/controllers/orders_controller.rb

```diff
@@ -1,12 +1,8 @@
 class OrdersController < ApplicationController
   def add_to_order
-    if Order.last
-      order = Order.last
-    else
-      order = Order.create
-    end
+    @order = Order.find_by(bought: false) || Order.create
```

Comment:
Also here, if you store your order info in a session you call pull it up when needed